addons/l10n_sa_edi/models/account_edi_format.py

```diff
@@ -1,15 +1,17 @@
 import json
-from hashlib import sha256
 from base64 import b64decode, b64encode
-from lxml import etree
 from datetime import date, datetime
-from odoo import models, fields, _, api
-from odoo.exceptions import UserError
-from cryptography.hazmat.primitives.serialization import load_pem_private_key
-from cryptography.hazmat.primitives.asymmetric.ec import ECDSA
-from cryptography.hazmat.primitives import hashes
+from hashlib import sha256
+
 from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric.ec import ECDSA
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
```

Comment:
@mebe-odoo Why does this need changing?

Reply:
@jco-odoo not sure why that changed, but should be good now